HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -907,19 +907,29 @@ def self.apply_lighting(hpxml)
     if hpxml.lighting.exterior_monthly_multipliers.nil?
       hpxml.lighting.exterior_monthly_multipliers = default_exterior_lighting_monthly_multipliers
     end
-    if (hpxml.lighting.holiday_exists == true) && hpxml.lighting.holiday_kwh_per_day.nil?
-      # From LA100 repo (2017)
-      if hpxml.building_construction.residential_facility_type == HPXML::ResidentialTypeSFD
-        hpxml.lighting.holiday_kwh_per_day = 1.1
-      elsif [HPXML::ResidentialTypeApartment, HPXML::ResidentialTypeSFA].include? hpxml.building_construction.residential_facility_type
-        hpxml.lighting.holiday_kwh_per_day = 0.5
-      end
-      hpxml.lighting.holiday_period_begin_month = 11
-      hpxml.lighting.holiday_period_begin_day_of_month = 24
-      hpxml.lighting.holiday_period_end_month = 1
-      hpxml.lighting.holiday_period_end_day_of_month = 6
-      hpxml.lighting.holiday_weekday_fractions = '0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.008, 0.098, 0.168, 0.194, 0.284, 0.192, 0.037, 0.019'
-      hpxml.lighting.holiday_weekend_fractions = '0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.008, 0.098, 0.168, 0.194, 0.284, 0.192, 0.037, 0.019'
+    if (hpxml.lighting.holiday_exists == true) 
+      if hpxml.lighting.holiday_kwh_per_day.nil?
+        # From LA100 repo (2017)
+        if hpxml.building_construction.residential_facility_type == HPXML::ResidentialTypeSFD
+          hpxml.lighting.holiday_kwh_per_day = 1.1
+        else # Multifamily and others
+          hpxml.lighting.holiday_kwh_per_day = 0.55
+        end
+      end
+      if hpxml.lighting.holiday_period_begin_month.nil?
+        hpxml.lighting.holiday_period_begin_month = 11
+        hpxml.lighting.holiday_period_begin_day_of_month = 24
```

Comment:
So if you had entered a begin day of month, this would get overwritten? You don't want to follow how run period does it, as a guide?

Reply:
I don't know if I understand your question correctly. 
So if only a _begin day of month_ was entered (without a _begin month_), the _begin day of month_ would get overwritten. Because EPvalidator requires all or none of a _begin month_ and _begin day of month_.  
https://github.com/NREL/OpenStudio-HPXML/blob/fa0cb26ecacb9519aa333502d34d31c3233b9410/HPXMLtoOpenStudio/resources/EPvalidator.rb#L837-L838
Therefore, checking whether a _begin month_ is nil in this if statement indicates whether both input variables are empty or provided. I thought I was following how _run period_ does. Am I missing something?